internal/api/mfa.go

```diff
@@ -35,6 +35,7 @@ type TOTPObject struct {
 type EnrollFactorResponse struct {
 	ID   uuid.UUID  `json:"id"`
 	Type string     `json:"type"`
+	FriendlyName string `json:"friendly_name"`
 	TOTP TOTPObject `json:"totp,omitempty"`
 }
```

Comment:
Have you run gofmt on this?

Reply:
Thanks! Not sure why [CI](https://github.com/supabase/gotrue/blob/master/.github/workflows/test.yml#L41) didn't error. Perhaps was running with old Go version